python/marvin/contrib/vacs/dapall.py

```diff
@@ -0,0 +1,67 @@
+# !usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under a 3-clause BSD license.
+#
+# @Author: Brian Cherinka
+# @Date:   2018-06-21 15:13:07
+# @Last modified by: José Sánchez-Gallego
+# @Last Modified time: 2018-07-07 14:14:55
+
+from __future__ import absolute_import, division, print_function
+
+import astropy
+from marvin.core.exceptions import MarvinError
+
+from .base import VACMixIn
+
+
+class DapVAC(VACMixIn):
+    """Provides access to the DAPall VAC.
+
+    VAC name: dapall
+
+    URL: https://www.sdss.org/dr15/manga/manga-data/catalogs/#DAPALLFile
+
+    Description: The DAPall file contains a per-galaxy summary of the DAP
+    output. This file is intended only as an example / template for other VACs,
+    and it is not included in Marvin as a VAC. DAPall data can be access in
+    Marvin using the .dapall attribute.
+
+    """
+
+    name = 'dapall'
+    path_params = {'dapver': '2.1.3', 'drpver': 'v2_3_1'}
```

Comment:
For the DAP that's easy because the versions are those that we use in Marvin anyway (the DAP is really a bad example, maybe we should not use it at all). But how are we supposed to do that for, for instance, the version of the Galaxy Zoo catalogue?

Reply:
I was also thinking about removing it since we have already included it, and it might be confusing for people.  But I'm flexible.  In my original idea, the user would have to download the vacs manually first.  They would then manually specify the parameters to the filepath.  Once the vac files were local, then they automatically get swept up into the container by Marvin.  But if you want the downloading to happen automatically we can discuss this more.

Reply:
If we require the VACs to have to be in the tree then I think there is little difference between the DAPall and galaxyzoo3d, so I think we can just remove the former and make it clearer.

I'm quite opposed to having to download the VAC manually. I think it defeats the purpose of easy access to VACs. I think we should link a release (MPL-6, DR15) to a VAC version but not go beyond that.

Reply:
I like linking VAC versions to a release.  That makes sense to me, especially since they are released for a specific release.  Then we can tell exactly what vacs became available when, plus we have some versioning which we can use to automatically download them.  We should build this into the DataModel for a specific release, so we know which VACs (and versions) are available in which release.